lib/node_modules/@stdlib/stats/base/cuminabs/test/test.ndarray.js

```diff
@@ -108,6 +108,74 @@ tape( 'the function calculates the cumulative minimum absolute value', function
 	t.end();
 });
 
+tape( 'the function calculates the cumulative minimum absolute value (accessor)', function test( t ) {
+	var expected;
+	var x;
+	var y;
+	var i;
+
+	x = [ 1.0, -2.0, 3.0, -4.0, 5.0 ];
+	y = [ 0.0, 0.0, 0.0, 0.0, 0.0 ];
+	cuminabs(x.length, toAccessorArray( x ), 1, 0, toAccessorArray( y ), 1, 0);
```

Comment:
```suggestion
	cuminabs( x.length, toAccessorArray( x ), 1, 0, toAccessorArray( y ), 1, 0 );
```

This applies here and below.